server/sonar-server/src/main/resources/org/sonar/server/computation/ws/CeProjectWsAction/example.json

```diff
@@ -0,0 +1,25 @@
+{
+  "queue": [
+    {
+      "id": "AU_w84A6gAS1Hm6h4_ih",
```

Comment:
I would put at least 2 (3 would be perfect) elements in the queue example to show there can be several elements.